schema/core/primitive/acyclic_graph.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
For brevity we can drop `acyclic` from the directory name and put in description

Reply:
Would also help with the "-" vs "_" in schema ids